packages/suite-desktop/src/modules/tor.ts

```diff
@@ -163,6 +163,22 @@ const load = async ({ mainWindow, store }: Dependencies) => {
         return { success: true };
     });
 
+    // Handle event emitted by request-interceptor module
+    let lastCircuitResetTime = 0;
+    mainThreadEmitter.on('module/reset-tor-circuits', event => {
+        const lastResetDiff = Date.now() - lastCircuitResetTime;
+        if (lastResetDiff > 30000) {
```

Comment:
Maybe having this number `30000` defined to a variable on the top like we have ` const host = '127.0.0.1';` with some meaningful name would help readability.